custom/conf/app.ini.sample

```diff
@@ -50,6 +50,11 @@ DISABLED_REPO_UNITS =
 DEFAULT_REPO_UNITS = repo.code,repo.releases,repo.issues,repo.pulls,repo.wiki
 ; Prefix archive files by placing them in a directory named after the repository
 PREFIX_ARCHIVE_FILES = true
+; Enable the kanban board feature system wide.
+ENABLE_KANBAN_BOARD = true
+; Default templates for kanban borards
+PROJECT_BOARD_BASIC_KANBAN_TYPE = Todo, In progress, Done
```

Comment:
```suggestion
PROJECT_BOARD_BASIC_KANBAN_TYPE = Todo, In Progress, Done
```
doesn't this look nicer?

Reply:
Undecided :)

Reply:
Undecided :)

Reply:
Adding to that, "Todo" isn't a single word either. Maybe change it to "To Do"?

Reply:
this are smal nits ... If there is no clear opinion just leave it as it is for now - this feature has to be added so maintaining will get easyer